controllers/util.go

```diff
@@ -24,6 +24,10 @@ import (
 
 func modelMeshEnabled(n *corev1.Namespace, controllerNamespace string) bool {
 	if v, ok := n.Labels["modelmesh-enabled"]; ok {
+		// Returns false if the namespace state is terminating even though the namespace have the 'modelmesh-enabled=true' label.
```

Comment:
This comment is helpful. Could you add comments to each of the changed code blocks above to describe what the code does and why (what problem it is solving).